Source: philantrop@exherbo.org
Upstream: Dead.
Reason: Fixes the displayed version.

diff -Naur arc-5.21o.orig/arc.c arc-5.21o/arc.c
--- arc-5.21o.orig/arc.c	2005-10-09 03:38:22.000000000 +0200
+++ arc-5.21o/arc.c	2008-09-28 17:38:16.000000000 +0200
@@ -129,7 +129,7 @@
 #endif
 
 	if (num < 3) {
-		printf("ARC - Archive utility, Version 5.21i, created on 11/25/92 at 14:40:55\n");
+		printf("ARC - Archive utility, Version 5.21o\n");
 /*		printf("(C) COPYRIGHT 1985,86,87 by System Enhancement Associates;");
 		printf(" ALL RIGHTS RESERVED\n\n");
 		printf("Please refer all inquiries to:\n\n");
diff -Naur arc-5.21o.orig/marc.c arc-5.21o/marc.c
--- arc-5.21o.orig/marc.c	2005-10-09 03:38:22.000000000 +0200
+++ arc-5.21o/marc.c	2008-09-28 17:38:30.000000000 +0200
@@ -68,7 +68,7 @@
 
 
     if(nargs<3)
-    {	 printf("MARC - Archive merger, Version 5.21i, created on 11/25/92 at 14:40:55\n");
+    {	 printf("MARC - Archive merger, Version 5.21o\n");
 /*	 printf("(C) COPYRIGHT 1985,86,87 by System Enhancement Associates;");
 	 printf(" ALL RIGHTS RESERVED\n\n");
 	 printf("Please refer all inquiries to:\n\n");
